playground/infrastructure/helper.py

```diff
@@ -199,7 +212,9 @@ def get_supported_categories(categories_path: str) -> List[str]:
     return yaml_object[TagFields.categories]
 
 
-def _get_example(filepath: str, filename: str, tag: dict) -> Example:
+def _get_example(
+    filepath: str, filename: str, tag: Dict[str, Union[str,
```

Comment:
This formatting is a bit ugly, could you please move `tag` to the next line?

Reply:
That's not my formatting :) yapf think that this one is beautiful :)